src/datetime/serde.rs

```diff
@@ -180,7 +188,10 @@ pub mod ts_nanoseconds {
     where
         S: ser::Serializer,
     {
-        serializer.serialize_i64(dt.timestamp_nanos())
+        serializer.serialize_i64(
```

Comment:
For all of these, why not yield errors here?

(Let's not have a `.` in `expect()` messages.)

Reply:
Good point, they should just return an error.